quint/src/parsing/quintParserFrontend.ts

```diff
@@ -202,6 +231,7 @@ export function parsePhase2sourceResolution(
           worklist.push([m, pathTrail.concat([importeePath])])
         })
         sourceToModules.set(importeePath.normalizedPath, newModules)
+        visitedPaths.set(importeePath.normalizedPath.toLocaleLowerCase(), importeePath.normalizedPath)
```

Comment:
1. Why are we using the locale version of `to*LowerCase` here?
2. Can we reuse the consts from above?
```suggestion
        visitedPaths.set(importeeLowerCase, importeeNormalized)
```